EncoreTickets.SDK/Pricing/IPricingServiceApi.cs

```diff
@@ -17,5 +19,58 @@ public interface IPricingServiceApi : IServiceApiWithAuthentication
         /// </summary>
         /// <returns>Exchange rates.</returns>
         ResponseForPage<ExchangeRate> GetExchangeRates(ExchangeRatesParameters parameters);
+
+        /// <summary>
+        /// Returns price bands for the specified product.
+        /// </summary>
```

Comment:
Please leave an endpoint that the method uses in a comment, this may be useful for external integrators